README_ID.md

```diff
@@ -0,0 +1,105 @@
+# [OFFICIAL] Member baru, Rekan kerja dan Tim -> Gabung disini !!! 🚀🚀🚀
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+[![README.md](https://img.shields.io/badge/English-up-brightgreen)](README.md)
+[![README_PT.md](https://img.shields.io/badge/Portuguese-up-brightgreen)](README_PT.md)
+[![README_ES.md](https://img.shields.io/badge/Spanish-up-brightgreen)](README_ES.md)
+
+![The 400](images/the-400.gif)
+
+## **Selamat Datang**,
+
+### kami adalah tim yang sedang berkembang dengan ide yang besar, dan kami membutuhkan bantuan anda.
+
+> Silahkan kirim pesan anda dalam bentuk issue baru jika anda tertarik, dan kami akan mengundundang anda ke dalam organisasi ini.
+
+**Yang tertarik atau orang baru, silahkan perkenalkan diri anda minimal dalam 2 baris kalimat.**
+
+**Posisi yang dibutuhkan:**
+
+- Programmers ruby
+- Programmer golang
+- Programmer python
+- Programmer lisp
+- Programmer rust
+- Programmer shell script
+- Front end developer
+- DevOps professional
+- System Admin
+- Tester
+- Writer / Penulis
+- Translators / Penerjemah
+- Designer
+- Graphic Designer
+- Community moderator
+- Community champion
+- Dan masih banyak lagi....
+
+**Kami sedang mengembangkan situs situs kami yang di host di:**
+
+- GitHub Pages
+- Heroku
+- Read the Docs
+- Cloudflare Workers
+- Surge
+- PythonAnywhere
+- Google Cloud
+- GitLab Pages
+- Kami juga berencana menambah penyedia hosting
+
+**Kami juga sedang bekerja di beberapa tool software, beberapa diantaranya adalah:**
+
+- DITA plugins
+- Command line tools
+- Web scrapers
+- Eclipse IDE based tools
+- XML / XSL / XML Schema tools
+- File differencing tools
+
+**Kami juga sedang mengembangankan penggunaan dari:**
+
+- GitHub Actions
+- CircleCI
+- TravisCI
+- Shell Script
+- Pre-commit hooks
+- GitHub Marketplace apps
+
+**Catatan:**
+
+Pengalaman dan skill kemampuan tidak dibutuhkan.
+
+Yang penting adalah mau untuk belajar dan selalu mencoba.
+
+Ya, itu semua yang dibutuhkan untuk memulai.
+
+Saya yakin, ini dapat menjadi pengalaman baru dengan banyak pengetahuan untuk ditingkatkan dan disebarkan.
+
+Terimakasih 👍 
+
+## Contributors ✨
```

Comment:
Please remove lines 82 downwards inclusive since this part is autogenerated and only appears on the English README.